content/python/concepts/modules/modules.md

```diff
@@ -13,15 +13,15 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-A module is a Python file that contains functions, definitions, and statements that can be included in an application.
+A **module** is a Python file that contains functions, definitions, and statements that can be included in an application. Technically, all `.py` files are modules.
 
 Think of a module as a code library that can be reused across applications. Modules improve code reusablity and allow us to access shared libraries and packages.
 
-Instead of entering commands directly into a Python interpreter, Python code can be saved as a module so that we can use it again later in other applications. Modules can also be used to access libraries such as `numpy`, `sklearn`, and the `date-time` built-in module.
+Instead of entering commands directly into a Python interpreter, code can be saved as a module for later use in other applications. Third-party libraries such as `numpy`, `sklearn`, and the `date-time` built-in library are all examples of modules.
```

Comment:
```suggestion
Instead of entering commands directly into a Python interpreter, code can be saved as a module for later use in other applications. Third-party libraries such as [`numpy`](https://www.codecademy.com/resources/docs/numpy), `sklearn`, and the `date-time` built-in library are all examples of modules.
```